modules/react/select/lib/SelectInput.tsx

```diff
@@ -61,28 +73,23 @@ export const SelectInput = createSubcomponent(TextInput)({
     model
   ) => {
     return (
-      <InputGroup data-width="ck-formfield-width">
+      <InputGroup data-width="ck-formfield-width" {...selectInputStencil()}>
         {inputStartIcon && model.state.selectedIds.length > 0 && (
-          <InputGroup.InnerStart data-part="select-start-icon-container" {...selectIconsStencil()}>
-            <SystemIcon data-part="select-start-icon" icon={inputStartIcon} />
+          <InputGroup.InnerStart {...selectInputStencil.parts.startIconContainer}>
+            <SystemIcon {...selectInputStencil.parts.startIcon} icon={inputStartIcon} />
```

Comment:
why data-part is not working?

Reply:
You mean it's not being applied to the element?

Reply:
so you replaced `data-part` attribute to stencil `selectInputStencil.parts.startIconContainer`
so i am kinda confused why, `data-part` seems easier to use

Reply:
if it's just data attribute, it seems overcomplicated